frontend/app/src/composables/api/task.ts

```diff
@@ -27,9 +28,10 @@ export function useTaskApi() {
     return handleResponse(response);
   };
 
-  const queryTaskResult = async <T>(id: number): Promise<ActionResult<T>> => {
+  const queryTaskResult = async <T>(id: number, transformer?: AxiosResponseTransformer[]): Promise<ActionResult<T>> => {
     const config: Partial<AxiosRequestConfig> = {
       validateStatus: validTaskStatus,
+      transformResponse: transformer ?? basicAxiosTransformer,
```

Comment:
no need to pass the basicTransformer here, either override with the custom one or let the defaults